### PR TITLE
Add repository for stale issue labeling for openshift-azure

### PR DESCRIPTION
**Pull request by troy0820**
* Add `repo:openshift/openshift-azure` to allow periodic job to run stale issues

### CODE REVIEW

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -158,6 +158,7 @@ periodics:
         --query=repo:openshift/origin
         repo:operator-framework/operator-sdk
         repo:tnozicka/openshift-acme
+        repo:openshift/openshift-azure
```
Comment (ehashman) [inline ci-operator/jobs/infra-periodics.yaml]: I think we also need to add openshift-azure to the -rotten and -close jobs.

Comment (troy0820) [reply ci-operator/jobs/infra-periodics.yaml]: That's fair, I didn't add it thinking we didn't want to automate that but I can push the commit to add it to the other two jobs 👍

Comment (ehashman) [reply ci-operator/jobs/infra-periodics.yaml]: Without all three jobs, this only achieves one portion of the closing out stale issues process :)

**Comment by ehashman** [comment]
/cc @bparees 

ARO team would like the openshift-bot to mark issues as stale/rotten/closed on the openshift-azure repo.

**Pull request merged**